docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,155 @@
+# Using an External Type
+
+There are several different external types that may be referenced when writing a controller.
+* Custom Resource Definitions (CRD) that is defined in the current project `kubebuilder create api`.
+* Core Kubernetes Resources eg. Deployments or Pods.
+* CRD's that are created and installed in another project.
+* A Custom Resource defined via an API Aggregation (AA). Aggregated APIs are subordinate APIServers that sit behind the primary API server, which acts as a proxy.
+
+Currently Kubebuilder handles the first two, CRDs and Core Resources, seamlessly.  External CRDs and CRs created via Aggregation must be scaffolded manually.
+
+In order to use a Kubernetes Custom Resource that has been defined in another project
+you will need to have several items of information.
+* The Domain of the CR
+* The Group under the Domain 
+* The Go import path of the CR Type definition.
+
+The Domain and Group variables have been discussed in other parts of the documentation.  The import path would be located in the project that installs the CR.
+
+This document uses `my` and `their` prefixes as a naming convention for repos, groups, and types to clearly distinguish between your own project and the external one you are referencing.
+
+Example external API Aggregation directory structure
+```
+github.com
+    ├── theiruser
+        ├── theirproject
+            ├── apis
+                ├── theirgroup
+                   ├── doc.go
+                   ├── install
+                   │   ├── install.go
+                   ├── v1alpha1
+                   │   ├── doc.go
+                   │   ├── register.go
+                   │   ├── types.go
+                   │   ├── zz_generated.deepcopy.go
+```
+
+In the case above the import path would be `github.com/theiruser/theirproject/apis/theirgroup/v1alpha1`
+
+### Create a project
+
+```shell
+kubebuilder init --domain $APIDOMAIN --owner "MyCompany"
+```
+
+### Add a controller
+
+be sure to answer no when it asks if you would like to create an api? [Y/n]
+```shell
+kubebuilder create api --group mygroup --version $APIVERSION --kind MyKind
+```
+
+## Edit the API files.
+
+### Register your Types
+
+Edit the following file to the pkg/apis directory to append their `AddToScheme` to your `AddToSchemes`:
```

Comment:
```suggestion
Note that the manager will only have the schemas of Kubernetes core types and those Types that are created and owned by your project. Therefore, since it is and third-party one you need to edit the following file to the pkg/apis directory to append their `AddToScheme` to your `AddToSchemes`:
```

Reply:
I think this is not what should happen. Usually you do not want to add external apis within your own apis. Thus you would rather call an additional Add To Scheme Method in your `main.go` instead of preadding one in apis

Reply:
Actually you already did that comment in https://github.com/kubernetes-sigs/kubebuilder/pull/3055#discussion_r1032751958 so im just gonna close this one here.